src/actions/commands/actions.ts

```diff
@@ -2140,7 +2140,7 @@ export class CommandYankFullLine extends BaseCommand {
   public async exec(position: Position, vimState: VimState): Promise<VimState> {
     const linesDown = (vimState.recordedState.count || 1) - 1;
     const start = position.getLineBegin();
-    const end = new Position(position.line + linesDown, 0).getLineEnd().getLeft();
+    const end = position.getDown(linesDown).getLeft();
```

Comment:
`getLeft()` might be necessary due to this line in `YankOperator`, which does no bounds-checking:
```let extendedEnd = new Position(end.line, end.character + 1);```

This could be refactored, but the PR currently seems like it's strictly improving matters, so I'll merge. Thank you!

Reply:
I understand, thanks for the confirmation and response!